qt/python/mantidqt/mantidqt/widgets/sliceviewer/model.py

```diff
@@ -140,7 +143,7 @@ def get_title(self, ws_name=None) -> str:
         of the model's workspace if none supplied.
         """
         if not ws_name:
-            ws_name = self.get_ws_name()
```

Comment:
Could I ask that we still use the accessor here? It means other parts of the code still don't need to know how the name is found just that there is a method to get at it.